### PR TITLE
store/tikv: move some tests into standalone directory

### PR DESCRIPTION
**Pull request by disksing**
Signed-off-by: disksing <i@disksing.com>

### What problem does this PR solve?
store/tikv has dependencies on `mockstore` and `unistore` because tests need to use them. We can resolve it by moving all tests to standaline package.

Part of https://github.com/pingcap/tidb/issues/22513

### What is changed and how it works?
What's Changed:
- move 3 test files into `store/tikv/tests`
- add *Probe utils to expose internal states of `store/tikv`.

### Check List 
Tests 
- Unit test

### Release note 
- No release note

### CODE REVIEW

**Review by AndreMouche — APPROVED** [review]
LGTM

File: store/tikv/probe.go [hunk 1]
```diff
@@ -0,0 +1,169 @@
+// Copyright 2021 PingCAP, Inc.
```
Comment (AndreMouche) [inline store/tikv/probe.go:1]: e, could we rename it like test_probe.go or something similar

Comment (disksing) [reply store/tikv/probe.go:1]: yes

**Review by AndreMouche — APPROVED** [review]

**Review by andylokandy — APPROVED** [review]

**Pull request merged**